.changeset/react.md

```diff
@@ -0,0 +1,37 @@
+---
+'@nhost/react': minor
+---
+
+New React client
+
+This release brings a full rewrite of the React client, to make it tree shakable and fully reactive.
+See the [documentation](https://docs.nhost.io/reference/react) for further information.
+
+### Hooks
+
+- `useAccessToken`
+- `useAnonymousSignIn`
+- `useAuthenticated`
+- `useAvatarUrl`
```

Comment:
Feels a bit verbose to have a hook just for the avatarUrl. How about a `useUser(Data)` hook with this, and more, user data?

Reply:
we could. But in terms of typings, userData is `User | null`, meaning devs won't be able to destructure.
so something like `const { email } = useUserData()` will raise a typescript error.
whereas `const email = useEmail()` won't.